optuna/progress_bar.py

```diff
@@ -75,15 +75,25 @@ def _init_valid(self) -> None:
         optuna_logging.disable_default_handler()
         optuna_logging._get_library_root_logger().addHandler(_tqdm_handler)
 
-    def update(self, elapsed_seconds: float) -> None:
+    def update(self, elapsed_seconds: float, best_trial: int, best_value: Optional[float]) -> None:
         """Update the progress bars if ``is_valid`` is :obj:`True`.
 
         Args:
             elapsed_seconds:
                 The time past since :func:`~optuna.study.Study.optimize` started.
+            best_trial:
+                The number of the current best trial.
+            best_value:
+                The score of the current best trial.
```

Comment:
`value` would be more consist with the terminology in Optuna docs.
```suggestion
                The value of the current best trial.
```